libs/wingc/src/jsify.rs

```diff
@@ -283,8 +283,25 @@ impl JSifier {
 			ExprKind::Reference(_ref) => self.jsify_reference(&_ref, None, phase),
 			ExprKind::Call { function, args } => {
 				let mut needs_case_conversion = false;
+				// note: Globals are emitted here and wrapped in "{{ ... }}" blocks. Wrapping makes these emissions, actual
+				// statements and not expressions. this makes the runtime panic if these are used in place of expressions.
```

Comment:
Isn't this already prevented by the type checker? I'm not sure I see the need to add a second safety mechanism.

e.g. If I write this:
```
let x = print("Hello, world!");
```

and compile it in Wing, it should error when type checking is performed (before any JS is emitted)

Reply:
You are correct. I just wanted to emit technically accurate code here, because spec does mention these are statements and not expressions. The block make these expression-statements at JS grammar level.

Reply:
From: https://github.com/tree-sitter/tree-sitter-javascript/blob/7a29d06274b7cf87d643212a433d970b73969016/grammar.js#L247